browser-facade/aspnetapp/Pages/Friends.cshtml

```diff
@@ -9,8 +9,8 @@
     @foreach (var conversation in Model.Conversations)
     {
         <li>
-            <a asp-page="/Messages" asp-route-receiver="@conversation.Receiver">
-                Conversation with @conversation.Receiver
+            <a asp-page="/Messages" asp-route-receiver="@conversation.Sender">
```

Comment:
FYI this may not work 50% of the case
See #56

Reply:
Why?